templates/rust/macros.tera

```diff
@@ -73,9 +106,10 @@ pub const fn {{reg.name~index| to_func_id }}(&self)-> crate::common::RegCore<{{r
 {%- endmacro -%}
 
 
-{%- macro register_struct(reg) -%}
-{%- set reg_struct_name = reg.name | to_struct_id -%}
-{%- set reg_mod_name = reg.name | to_mod_id -%}
+{%- macro register_struct(reg_name,reg) -%}
```

Comment:
It would be better to add reg_name at the end of the parameters and use default value. see https://keats.github.io/tera/docs/#macros
This would be better because in the most of the cases specifing the reg_name is not necessary

Reply:
Done